environment/deployments/science-platform/main.tf

```diff
@@ -176,6 +176,19 @@ module "service_account_cluster" {
   ]
 }
 
+moved {
+  # Google provider version update
+  from = module.service_account_cluster.google_service_account.service_accounts["cluster"]
+  to   = module.service_account_cluster.google_service_account.service_accounts[0]
+}
+
+moved {
+  # Google provider version update
+  from = module.service_account_cluster.google_project_iam_member.project-roles["cluster-science-platform-dev-7696=>roles/container.clusterAdmin"]
```

Comment:
The`cluster-science-platform-dev-7696` service account is unique per project.  Please add the service accounts from demo, int, and stable, too.  In the plans the service account membership in `clusterAdmin` is removed.

Reply:
Turns out we're gonna have to edit the state directly to do this.